torch/csrc/jit/fusion_compiler.cpp

```diff
@@ -625,8 +625,13 @@ struct DynamicLibrary {
 static const std::string so_template = "/tmp/pytorch_fuserXXXXXX.so";
 static const std::string cpp_template = "/tmp/pytorch_fuserXXXXXX.cpp";
 
+#ifdef __PPC64__ 
+static const std::string compile_string =
+  "\"${cxx}\" -O3 -g -std=c++11 -fPIC -shared \"${cpp_file}\" -o \"${so_file}\"";
+#else
 static const std::string compile_string =
   "\"${cxx}\" -O3 -g -march=native -std=c++11 -fPIC -shared \"${cpp_file}\" -o \"${so_file}\"";
+#endif
```

Comment:
It would be nicer to split it off like this:
```cpp
static const std::string compile_string =
  "${cxx} -O3 ... "
#ifndef __PPC64__
  " -march=native "
#endif
  " ... ${so_file}"
```